main/src/org/destinationsol/SolApplication.java

```diff
@@ -250,7 +248,8 @@ public OggSoundManager getSoundManager() {
         return soundManager;
     }
 
-    public void paused() {
+    // TODO: Why do we even have this method? Look into it's removal.
+    public void pauseApplication() {
```

Comment:
Guessing it has to do either with the window losing focus or the in-game menu being used?